pmm-app/src/onboard/components/ActionCard/ActionCard.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FC } from 'react';
+import {
+  Card, HorizontalGroup, IconButton, useStyles2,
+} from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+
+interface ActionCardProps {
+  onClick?: () => void;
+  hasExternalLink?: boolean;
+  onExternalLinkClick?: () => void;
+  externalLinkTooltip?: string,
+  variant: ActionCardVariant,
+  imgSrc?: string,
+  imgAlt?: string,
+  heading: string,
+  description?: string,
+}
+
+export enum ActionCardVariant {
+  Primary,
+  Secondary,
+}
+
+export const ActionCard: FC<ActionCardProps> = ({
+  hasExternalLink,
+  imgSrc,
+  variant,
+  onClick,
+  onExternalLinkClick,
+  heading,
+  description,
+  externalLinkTooltip,
+  imgAlt,
+}) => {
+  const styles = useStyles2(getStyles);
+
+  const containerClass = variant === ActionCardVariant.Primary
+    ? styles.actionCardLogoContainerPrimary
+    : styles.actionCardLogoContainerSecondary;
+
+  return (
+    <div className={styles.actionCard} onClick={onClick}>
+      <div className={containerClass}>
+        <img className={styles.actionCardImg} src={imgSrc} alt={imgAlt} />
+      </div>
+      <Card>
+        <Card.Heading>
+          <HorizontalGroup justify="space-between">
+            <Card.Heading className={styles.cardHeading}>{heading}</Card.Heading>
+            <Card.SecondaryActions>
+              {hasExternalLink && (
+                <IconButton
+                  name="external-link-alt"
+                  tooltip={externalLinkTooltip}
+                  className={styles.externalLinkIcon}
+                  size="lg"
+                  onClick={(e) => {
+                    if (onExternalLinkClick) {
+                      e.stopPropagation();
+                      onExternalLinkClick();
+                    }
+                  }}
+                />
+              )}
+            </Card.SecondaryActions>
+          </HorizontalGroup>
+        </Card.Heading>
+        <Card.Description className={styles.cardDescription}>{description}</Card.Description>
+      </Card>
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  cardHeading: css`
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 21px;
+    color: ${theme.colors.text.primary};
+  `,
+  cardDescription: css`
+    min-height: ${theme.typography.body.lineHeight}em
+  `,
+  externalLinkIcon: css`
+    height: 16.67px;
+    width: 16.67px;
+    margin-right: 0 !important;
```

Comment:
do we need this? to use !important rule you should have good reason for this, as I know people try to avoid using that.

Reply:
yeah, it is unfortunately set as `!important` in grafana, so to align the icon appropriately had to override it